### PR TITLE
Adds interactive mobs to Centcomm evac when the game ends

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl:
add: Centcom employees have finally returned after their "vacations". They may look completely different, but they are the same employees!
/:cl:

Funny how NT's command capitol is always B A R R E N

### CODE REVIEW

**Comment by Incoming5643** [comment]
will endlessly conflict with admins spawning things/mobs on the shuttle dock.

**Comment by Cyberboss** [comment]
Just varedit the number to zero then

**Comment by Cyberboss** [comment]
I'll make it easier, hold on

File: code/controllers/subsystem/npcpool.dm [hunk 1]
```diff
@@ -124,4 +131,54 @@ var/datum/controller/subsystem/npcpool/SSnpc
...
+
+	for(var/I in 1 to NPCsToSpawn)
+		var/list/candidates = get_area_turfs(pick(centcom_areas))
+		if(!LAZYLEN(candidates))
+			break
+		
+		while(candidates.len)
+			var/turf/T = pick(candidates)
+			if(!is_blocked_turf(T) && !spawn_locs[T])
+				spawn_locs[T] =
```
Comment (RemieRichards) [inline code/controllers/subsystem/npcpool.dm]: excellent

**Comment by Bawhoppen** [comment]
This seems quite pointless.

**Comment by Cheridan** [comment]
Fuckhuge Centcom is already problematic, every time I spawn an inspector they get completely lost and I have to warp them to the shuttle.

**Comment by Cyberboss** [comment]
These guys don't spawn until the round is officially over and can be disabled by admins if they choose. Also they won't even eat any CPU once #25429 is merged.

**Comment by ExcessiveUseOfCobblestone** [comment]
why was centcomm made big anyways? It's not meant to be a place you go to casually, it's meant for you to hang around for a minute in before the round resets.

**Comment by Shadowlight213** [comment]
@ExcessiveUseOfCobblestone No clue. We've wanted a bigger centcom for a while, but okand went overboard

**Comment by KorPhaeron** [comment]
This isn't popular among admins

**Pull request closed without merging**